com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -940,13 +852,69 @@ private void OnDestroy()
             }
         }
 
+        /// <summary>
+        /// Global shtudown. Disconnects clients and stops server.
+        /// </summary>
         public void Shutdown()
         {
             if (NetworkLog.CurrentLogLevel <= LogLevel.Developer)
             {
                 NetworkLog.LogInfo(nameof(Shutdown));
             }
 
+            if (IsServer)
+            {
+                var disconnectedIds = new HashSet<ulong>();
+
+                //Don't know if I have to disconnect the clients. I'm assuming the NetworkTransport does all the cleaning on shtudown. But this way the clients get a disconnect message from server (so long it does't get lost)
```

Comment:
some grammar mistakes there and also in the summary section.

+ I'd like to get @andrews-unity quick thoughts about this PR :)

Reply:
(also the comment should not exist and merged in, probably)

Reply:
I moved what was in the old Stop methods, I will clean it up a bit.

Reply:
I think the comment should **continue** to exist. Because it's a quirk of the current transport implementation. Perhaps rephrased.